website/content/docs/nia/enterprise/license.mdx

```diff
@@ -57,15 +67,15 @@ Licenses have an expiration date and a termination date. The termination date is
 
 The time between the expiration and termination dates is a grace period. Grace periods are generally 24-hours, but you should refer to your license agreement for complete terms of your grace period.
 
-When approaching expiration and termination, CTS Enterprise will provide notifications in the system logs:
+When approaching expiration and termination, by default, CTS Enterprise will attempt to retrieve a new license. If auto-retrieval is disabled, CTS Enterprise will provide notifications in the system logs:
 
-| Time period                                 | Behavior                          |
-| ------------------------------------------- | --------------------------------- |
-| 30 days before expiration                   | Warning-level log every 24-hours  |
-| 7 days before expiration                    | Warning-level log every 1 hour    |
-| 1 hour before expiration                    | Warning-level log every 1 minute  |
-| 1 day before expiration                     | Warning-level log every 5 minutes |
-| At or after expiration (before termination) | Error-level log every 1 minute    |
-| At or after termination                     | Error-level log and exit          |
+| Time period                                 | Behavior - auto-retrieval enabled (default) |Behavior - auto-retrieval disabled |
+| ------------------------------------------- |-------------------------------------------- |---------------------------------- |
+| 30 days before expiration                   | License retrieval attempt every 24-hours    |Warning-level log every 24-hours   |
+| 7 days before expiration                    | License retrieval attempt every 1 hour      |Warning-level log every 1 hour     |
+| 1 hour before expiration                    | License retrieval attempt every 1 minute    |Warning-level log every 1 minute   |
+| 1 day before expiration                     | License retrieval attempt every 5 minute    |Warning-level log every 5 minutes  |
```

Comment:
This is my bad - looks like line 76 and 77 are swapped i.e. the row for 1 hour should be after 1 day - this is unrelated to your PR and likely something I did a while back. Would you mind swapping these two rows as part of this PR? 🙏

Reply:
Minor typo: missing "s" in "5 minutes" in the license retrieval column